user_management_backend/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM ruby:2.6.3
+RUN wget -qO- https://deb.nodesource.com/setup_10.x | bash -
+RUN apt-get install -y build-essential libpq-dev nodejs
```

Comment:
why's nodejs involved in this build step?

Reply:
I see, I was following the instructions on the documentation. From my understanding, node.js is required for the asset pipeline to concatenate and minify assets(CSS, JS) for a rails application. Since this is a rails api backend, this should probably be removed as we are only exposing api endpoints.